spotipy/oauth2.py

```diff
@@ -213,7 +222,7 @@ def parse_response_code(self, url):
     def _make_authorization_headers(self):
         return _make_authorization_headers(self.client_id, self.client_secret)
 
-    def get_access_token(self, code):
+    def request_access_token(self, code):
```

Comment:
This looks like a public method so if possible I would avoid renaming it

Reply:
Yes, you're right about that. I intended to add another `get_access_token` method which would just return the possibly refreshed token, but found that `get_cached_token` is basically doing this exact thing. I removed this change from the commit and force-pushed an update.